_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
```

Comment:
Any reason not to skip a version with machines, in order to keep in sync?

Reply:
There is not any reason to keep in sync actually, these are separate projects. Podman is in release 31 for example.

Reply:
All the more reason to add the various versions to the about dialog... https://github.com/cockpit-project/cockpit/issues/15962